components/sankey/elements/title.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Title
+page_title: Sankey Title
+description: Title of the Sankey Diagram for Blazor.
+slug: sankey-title
+tags: telerik,blazor,sankey,diagram,chart,title
+published: True
+position: 7
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Sankey Title
+
+The Sankey diagram for Blazor allows you to specify a title for the diagram. This article explains how to add and configure the title.
+
+## Adding a Title
+
+1. Declare a `<SankeyTitle>` tag as a direct child of `<TelerikSankey>`.
+1. Set the `Text` parameter of the `<SankeyTitle>` tag.
+
+You can customize the title through the parameter of the `<SankeyTitle>` tag and through the nested tags it exposes.
+
+### Basic Customization
+
+The `<SankeyTitle>` tag exposes the following parameters for customization of the title:
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --------- | ---- | ----------- |
+| `Align` | `SankeyTitleAlign` enum | The alignment of the title. |
+| `Background` | `string`  | The background color of the title. |
+| `Color` | `string`  | The text color of the title. |
+| `Description` | `string`  | The accessible description of the Sankey. Added as `aria-label` to the `<div class="k-sankey">` element. |
+| `Font` | `string`  | The font of the title. |
+| `Position` | `SankeyTitlePosition` enum | The position of the title. |
+| `Visible` | `bool?` <br/> (`true`) | Whether the title is visible.|
+
+### Nested Customization Tags
+
+The `<SankeyTitle>` tag exposes nested tags for further customization. The structure of the nested tags is `<SankeyTitle*Specifics*>`, where the specifics can be:
+
+* [`Border`](/blazor-ui/api/telerik.blazor.components.sankeytitleborder)
+* [`Margin`](/blazor-ui/api/telerik.blazor.components.sankeytitlemargin)
+* [`Padding`](/blazor-ui/api/telerik.blazor.components.sankeytitlepadding)
+
+## Example
+
+Example customization the Sankey title by using nested tag settings.
```

Comment:
Ideally, all example intros would use consistent wording and structure. This one differs from the rest.

Re: the discussion around captions. My observation is that we're going with captions when the example is section/feature-related, and we're going with an Example section when the example covers pretty much everything covered by the article.